packages/kbn-search-api-panels/components/ingest_data.tsx

```diff
@@ -21,7 +21,12 @@ interface IngestDataProps {
   codeSnippet: string;
   selectedLanguage: LanguageDefinition;
   setSelectedLanguage: (language: LanguageDefinition) => void;
-  docLinks: any;
+  docLinks: {
+    beats: string;
+    connectors: string;
+    integrations: string;
+    logStash: string;
```

Comment:
Nitpick
```suggestion
    logstash: string;
```

Reply:
I don't disagree, but I'll fix this in a follow-up as this key is already in use and actually set from serverless search